src/main/webapp/js/App.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * module controls logic for Trivia api form submission,
+ * @returns {boolean}
+ * @constructor
+ */
+export const TriviaApi = () => {
+    event.preventDefault();
+
+    // get search form from dom
+    const searchForm = document.getElementById("form-api").elements;
+
+    // create url with params from form elements
+    const url = "/TrivialTrivia/api/trivia/request/?" + $.param({
+        triviaAmount: searchForm.namedItem("triviaAmount").value,
+        triviaCategory: searchForm.namedItem("triviaCategory").value,
+        triviaDifficulty: searchForm.namedItem("triviaDifficulty").value,
+        triviaType: searchForm.namedItem("triviaType").value,
+    });
+
+    // // send request for trivia, pass results to results showResults
+    // fetch(url, {headers: {'Content-type': 'application/json; charset=UTF-8', 'Accept': 'application/json'}})
+    //     .then(response => response.json()) // parse response as json
+    //     .then(data => showResults(data))// pass data to call back function
+    //     .catch(error => alert("Error sending request, try again." + " url: " + url));
```

Comment:
I'm confused, for some reason I'm seeing two version of all the files that you edited- but I commented on this in the first version I saw.